engine/lib/membrane_rtc_engine/engine.ex

```diff
@@ -1174,6 +1250,16 @@ defmodule Membrane.RTC.Engine do
     |> Map.get(track_id)
   end
 
+  defp prepare_track_notifications(state, track_id, notification) do
```

Comment:
I would go even further and pass here `subscriptions` instead of `state`. This way we make the function signature even more clearer